### PR TITLE
fix: user should be able to quickly rename requests by clicking [INS-3250]

### PR DESCRIPTION
**Pull request by ihexxa**
changelog(Improvements): Added a quicker way for users to quickly rename requests by clicking on them

Changes:
- enable quick renaming request by clicking

### CODE REVIEW

File: packages/insomnia/src/ui/components/editable-input.tsx [hunk 1]
```diff
@@ -6,11 +6,13 @@ export const EditableInput = ({
   value = 'Untitled',
   ariaLabel,
   name,
+  paddingClass,
   onChange,
 }: {
   value: string;
   ariaLabel?: string;
   name?: string;
+    paddingClass?: string;
```
Comment (ihexxa) [inline packages/insomnia/src/ui/components/editable-input.tsx]: The formatter seems always inserts incorrect indent, checking.

**Comment by jackkav** [comment]
Since double click is [not encouraged ](https://github.com/adobe/react-spectrum/discussions/1585?sort=new)with react-aria.
This PR is about making two mechanics play nicely.
- navigate
- rename

There seem to be only two approaches
1. clicking the text triggers a rename
2. clicking an edit icon triggers the rename

Problems:
option 1 can interfere with navigation
option 2 can reduce the space available to show the name

This PR appears to be applying option 1. I would encourage some group testing to address the problem with option 1 before approving/merging this PR.
Thanks.

**Comment by nickvruiz** [comment]
Please revert this, it is frustrating.

**Comment by guilhermeferrari** [comment]
> Please revert this, it is frustrating.

+1. At least give us the option to disable 😭

**Comment by bonetou** [comment]
Please revert this, it is frustrating 2.

**Comment by AntoRin** [comment]
There is no way to know you are already in an edit mode, either. I'd rather have clicks used for navigation between requests. A double-click can open up the rename dialog.

**Comment by alisonsm92** [comment]
I'm having a bad experience after this modification. I think the clicks should be used to navigate the list of requests. 
A double click can be used to rename.

**Comment by r1cardosouza** [comment]
bad idea. please, revert this!!!!!

**Comment by thedomeffm** [comment]
I thought it is a bug. Its super annoying.

**Pull request merged**